integration/truncindex_test.go

```diff
@@ -0,0 +1,133 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	runtimeapi "k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime"
+)
+
+func genTruncIndex(normalName string) string {
+	return normalName[:3]
+}
+
+func TestTruncIndex(t *testing.T) {
+	t.Logf("Pull an image")
+	const appImage = "busybox"
+	img, err := imageService.PullImage(&runtimeapi.ImageSpec{Image: appImage}, nil)
+	require.NoError(t, err)
+	imgTruncId := img[:len(img)/2]
+	defer func() {
+		assert.NoError(t, imageService.RemoveImage(&runtimeapi.ImageSpec{Image: imgTruncId}))
+	}()
+
+	t.Logf("Get image status by truncindex, truncId: %s", imgTruncId)
+	res, err := imageService.ImageStatus(&runtimeapi.ImageSpec{Image: imgTruncId})
+	require.NoError(t, err)
+	require.Equal(t, true, (res != nil) && (res.Id != ""))
+	t.Logf("Image status: %v", res)
```

Comment:
This will be spammy, don't need to log out.

Reply:
done